### PR TITLE
Make non-default targets (aa-symex etc) actually build

### PR DESCRIPTION
**Pull request by tautschnig**
This is the necessary follow-up to #446. It also reverts the non-PR-ed changes to the Travis config.

### CODE REVIEW

**Comment by kroening** [comment]
Is aa-symex still used by anyone?

**Comment by tautschnig** [comment]
I have no idea about the use of aa-symex, i.e., I'm not sure it ever was "used" - clearly there are differences between aa-symex and symex, but I have no idea which of them is "better"? This commit is to make those differences as explicit as possible.

File: src/aa-symex/Makefile [hunk 1]
```diff
@@ -11,8 +11,15 @@ OBJ += ../ansi-c/ansi-c$(LIBEXT) \
        ../solvers/solvers$(LIBEXT) \
        ../util/util$(LIBEXT) \
        ../goto-symex/adjust_float_expressions$(OBJEXT) \
+       ../goto-symex/rewrite_union$(OBJEXT) \
        ../pointer-analysis/dereference$(OBJEXT) \
-       ../aa-path-symex/aa-path-symex$(LIBEXT)
+       ../goto-instrument/cover$(OBJEXT) \
+       ../aa-path-symex/aa-path-symex$(LIBEXT) \
+			 ../symex/symex_main$(OBJEXT) \
```
Comment (peterschrammel) [inline src/aa-symex/Makefile]: weird indentation

**Comment by peterschrammel** [comment]
@tautschnig, tests are crashing here: https://travis-ci.org/diffblue/cbmc/jobs/192947737#L1231

**Comment by tautschnig** [comment]
Regarding crashes: That may be a consequence of e527f64 - taking a look.

**Comment by tautschnig** [comment]
Fixed in #467.

**Comment by tautschnig** [comment]
Space oddities addressed.

**Pull request merged**